File: backend/tiggle-root/tiggle/src/main/java/com/side/tiggle/domain/txtag/repository/TxTagRepository.java
package com.side.tiggle.domain.txtag.repository;

import com.side.tiggle.domain.txtag.model.TxTag;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface TxTagRepository extends JpaRepository<TxTag, Long> {

    Optional<TxTag> findByTxId(Long txId);
}

File: backend/tiggle-root/tiggle/src/main/kotlin/com/side/tiggle/domain/txtag/repository/TxTagRepository.kt
package com.side.tiggle.domain.txtag.repository

import com.side.tiggle.domain.txtag.model.TxTag
import org.springframework.data.jpa.repository.JpaRepository

interface TxTagRepository: JpaRepository<TxTag, Long> {
    fun findByTxId(txId: Long?): TxTag?
}